docs/architecture/adr-020-protobuf-transaction-encoding.md

```diff
@@ -105,20 +105,24 @@ message SignerInfo {
     // ModeInfo describes the signing mode of the signer and is a nested
     // structure to support nested multisig pubkey's
     ModeInfo mode_info = 2;
+    // account_sequence is the sequence of the account, which describes the
+    // number of transactions sent from a given address. It is used to prevent
```

Comment:
```suggestion
    // number of transactions signed by a given address. It is used to prevent
```

Reply:
While "sent from" might not be perfect, I still believe it's clearer than "signed by". I can sign 100 transactions, and never broadcast them.

Reply:
And I can sent multisig transactions I never signed (I have done this a few time IRL). How about:

`// number of committted transactions signed by a given address`

"committed" should be understood to be in a block (tendermint commits the block), but maybe there is a better word